test/dlc_tests/benchmark/sagemaker/mxnet/training/resources/mx_sm_benchmark.py

```diff
@@ -33,7 +33,7 @@
     role="SageMakerRole",
     train_instance_count=args.node_count,
     train_instance_type=args.instance_type,
-    image_name=args.image_uri,
+    image_uri=args.image_uri,
```

Comment:
This API upgrade to sagemaker v2 seems to be partial.

Also, the `pip install sagemaker` for this test is located at https://github.com/aws/deep-learning-containers/blob/master/test/test_utils/__init__.py#L630

Reply:
Updated pkg version.